mkosi.profiles/splash/mkosi.conf

```diff
@@ -0,0 +1,6 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+[Content]
+KernelCommandLine=quiet splash plymouth.use-simpledrm
```

Comment:
It would probably be best to make this append to the cmdline instead of overriding. Is that a thing in mkosi?

Reply:
We append by default to list based settings like this one